godot-core/src/builtin/color.rs

```diff
@@ -294,6 +298,53 @@ impl Color {
         ]))
     }
 
+    /// Convert `Color` into [`ColorHsv`].
+    ///
+    /// ## Panics
+    ///
+    /// Method will panic if the RGBA values are outside of the valid range `0.0..=1.0`. You can use [`Color::normalized`] to ensure that
+    /// they are in range, or use [`Color::try_to_hsv`].
+    #[must_use]
```

Comment:
We only use this attribute if there is potential ambiguity, e.g. in `Rect2::grow()`.

`to_*` methods are a well-known Rust convention for conversions, so it's very unlikely that someone would call this and ignore the return type. In fact there are even lints if you declare a `to_*` method that does something else.